src/applications/vaos/services/appointment/transformers.js

```diff
@@ -1,17 +1,41 @@
-import moment from '../../utils/moment-tz';
+import moment from '../../lib/moment-tz';
 
 import {
   APPOINTMENT_STATUS,
   APPOINTMENT_TYPES,
-  CANCELLED_APPOINTMENT_SET,
-  FUTURE_APPOINTMENTS_HIDE_STATUS_SET,
-  PAST_APPOINTMENTS_HIDE_STATUS_SET,
   PURPOSE_TEXT,
   EXPRESS_CARE,
   UNABLE_TO_REACH_VETERAN_DETCODE,
 } from '../../utils/constants';
 import { getTimezoneBySystemId } from '../../utils/timezone';
 
+export const CANCELLED_APPOINTMENT_SET = new Set([
```

Comment:
I think if we need to export these, I'd leave them in the index.js file in this folder. It seems odd to me to have them exported from here, since it's just static appointment information.